web/src/config/client_swagger.json

```diff
@@ -214,11 +163,20 @@
 					"201": {
 						"description": "User has been created"
 					},
-					"404": {
-						"description": "No Data Found"
-					},
 					"400": {
-						"description": "Bad Request"
+						"description": "Bad Request: Request body is missing or Invalid arguments"
+					},
+					"401": {
+						"description": "Unauthorized: Token has not found or invalid format or session is invalid"
+					},
+					"422": {
```

Comment:
409 can appear when trying to create duplicate user

Reply:
removed 409 from other places except user, s3 use and Iam user